src/_data/team.yml

```diff
@@ -926,4 +926,15 @@
     <p>He inherited from his past experience, in other contexts, a set of soft skills such as a natural ability to comprehend the business domain and the business needs, negotiate solutions, understand different perspectives or mitigate contrasts.</p>
     <p>In his free time he loves cycling, swimming, playing drums, cooking, reading and listening to jazz music.</p>
   email: riccardo.toni@codurance.com
-  office: london
\ No newline at end of file
+  office: london
+
+- id: jose_rh
+  name: José E Rodriguez
+  image: /assets/custom/img/team/jose_e_rodriguez.jpg
+  role: Managing Director (Spain)
+  fullDescription:
+    <p>I lead and support a team of exceptional people whose purpose is to accelerate organizations and to help them reach their goals. Together we help organizations tackle the challenges they face, guiding and mentoring them as they evolve by improving how they work.</p>
+
+  twitter: jrhuerta
+  github:
```

Comment:
We should remove the github tag if it is empty

Reply:
Solved, thanks